src/lib/jar.js

```diff
@@ -73,6 +73,21 @@ var Jar = {
         var expires = '; expires=' + new Date(new Date().setYear(new Date().getFullYear() + 1)).toUTCString();
         var path = '; path=/';
         document.cookie = obj + expires + path;
+    },
+    getSignedValue: function (cookieName, signedValue, callback) {
```

Comment:
Maybe `getAndUnsign` or just `getUnsignedValue` since `get` already implies it's just going to return the value as-is?  I think for consistency `signedValue` should become `name` like in the other methods.